nwbwidgets/utils/timeseries.py

```diff
@@ -128,7 +128,7 @@ def timeseries_time_to_ind(node: TimeSeries, time, ind_min=None, ind_max=None) -
             kwargs.update(lo=ind_min)
         if ind_max is not None:
             kwargs.update(hi=ind_max)
-        return bisect(node.timestamps, time, **kwargs)
+        return bisect(node.timestamps, time, **kwargs)-1
```

Comment:
again,
for timestamps=[0,1,2,3,4,5]
the index at time=5 should be 5 but bisect will return 6 since it will find the location where if 5 is inserted, will keep the list in sorted order.

Reply:
look at method `bisect_left`

Reply:
okay, i've made a change here: https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets/pull/152/commits/e2a0abb5be07d46fba48a8fc27027954c703dded